tests/locallib_test.php

```diff
@@ -202,10 +202,10 @@ public function test_pick_questions() {
         $this->assertEquals($questions, tool_securityquestions_pick_questions($USER));
 
         // Set period of questions to 5 seconds.
-        set_config('questionduration', 5, 'tool_securityquestions');
+        set_config('questionduration', 1, 'tool_securityquestions');
 
         // Wait 5 seconds to ensure fresh question choice.
-        sleep(5);
+        sleep(2);
```

Comment:
I think much better to move to the time() out and in as an argument:

tool_securityquestions_lock_user($user, $time)